ios/MullvadVPN/SettingsManager/SettingsManager.swift

```diff
@@ -153,6 +163,48 @@ enum SettingsManager {
         }
     }
 
+    /// Removes all legacy settings, device state and tunnel settings but keeps the last used
+    /// account number stored.
+    static func resetStore(completely: Bool = false) {
+        logger.debug("Reset store.")
+
+        do {
+            try store.delete(key: .deviceState)
+        } catch {
+            if (error as? KeychainError) != .itemNotFound {
```

Comment:
I believe the checks are here in order to keep logs somewhat less crowded or misleading (since trying to delete a non-existing entry isn't really an error). What does @pronebird think?

Reply:
Keychain throws errors when trying to delete an item that does not exist, so we ignore them here by checking against `.itemNotFound` error.